src/pages/Send/components/RecipientPicker/MultiAddressWalletItem/MultiAddressWalletItem.js

```diff
@@ -19,13 +19,15 @@ export function MultiAddressWalletItem({
   isOpen,
 }: Props) {
   return (
-    <div className={itemStyles.core}>
-      <JIcon name='wallet-use-fill' color='gray' className={itemStyles.icon} />
-      <div className={itemStyles.wrap}>
-        <span className={itemStyles.title}>{title}</span>
-        <span className={itemStyles.description}>{t`${addressCount} addresses`}</span>
+    <div className={multiAddressWalletItemStyles.core}>
+      <JIcon name='wallet-use-fill' color='gray' className={multiAddressWalletItemStyles.icon} />
+      <div className={multiAddressWalletItemStyles.wrap}>
+        <span className={multiAddressWalletItemStyles.title}>{title}</span>
+        <span className={multiAddressWalletItemStyles.description}>
+          {t`${addressCount} addresses`}
```

Comment:
You should use [`ngettext`](https://ttag.js.org/docs/ngettext.html) here, because in other languages you would have other rules for numbers. Like:
- 3 адрес**а**
- 5 адрес**ов**